pkg/cmd/server/origin/master.go

```diff
@@ -935,16 +935,6 @@ func (c *MasterConfig) RunDeploymentImageChangeTriggerController() {
 	controller.Run()
 }
 
-func (c *MasterConfig) RunDeploymentCancellationController() {
-	kclient := c.DeploymentCancellationControllerClient()
-	factory := deployconfigcontroller.DeploymentConfigControllerFactory{
```

Comment:
@abhgupta Doesn't this mean that instead of starting the DeploymentCancellationController in master we're starting a second DeploymentConfigController?